### ISSUE TITLE
Proxy unresponsive from iPhone

### ISSUE THREAD

**Issue by rmischook** [ProxymanApp/Proxyman#874]
### Proxyman version? (Ex. Proxyman 1.4.3)
2.24.0 (22400)

### macOS Version? (Ex. mac 10.14)
11.3
### Steps to reproduce 
1. proxy iPhone to Proxyman running on same wifi network
2. Go to any website on Safari
### Expected behavior
1. Expect to see IP address of the iPhone turn up in Proxyman sidebar under REMOTE DEVICES immediately with list of domains it has connected to
### Actual behaviour
It can take minutes for the expected to occur. Once it happens it works fine but a period of inactivity can result in the proxy not responding. Have tried restarting proxy but doesn't seem to have an effect. Older versions of Proxyman seem less susceptible to the issue.

**Comment by NghiaTranUIT** [comment]
Hi @rmischook, Proxyman hasn't changed the logic regarding Remote Proxy from iOS Devices.

There are many external sources that might cause this problem. Next time, if you encounter this problem, let try:

1. Double check whether or not you're using any VPNs on your iOS Device. If yes, please turn it OFF.
2. Double check the Firewall setting in your macOS System Preference -> Security -> Privacy tab -> Turn it off because it might block all incoming connections.

Here is the a troubleshooting: https://docs.proxyman.io/troubleshooting/my-ios-devices-couldnt-connect-to-proxyman-via-proxy

**Comment by rmischook** [comment]
Hi. I tried the troubleshooting but none applied, i.e. firewall not running and no VPN on my Mac nor my iPhone. It's a weird bug because the thing will work fine for hours and then - just stop. It's happened just now and here are some of the steps I've taken:

1. Disconnect iPhone from Proxy and reconnect.
2. Restart the phone
3. Restart Proxyman
4. Add another iPhone to proxy -> not working either
5. Turn on LOCAL proxying (I usually have that off as I don't want to proxy my Mac - just the iPhone). When I enable LOCAL I can see the traffic from the Mac going through 
6. Quit Proxyman and wait for awhile (at least 5 minutes). This fixed it just now.

I get this is hard to fix if you can't replicate but maybe someone else is seeing this. I'm on a home network using BT standard equipment for a fibre-to-the-home installation. No funky config on my home network.

**Comment by NghiaTranUIT** [comment]
Thanks for the reply @rmischook 

Not sure what causes your problem. I will collect this bug from other user and try to reproduce it on our network 👍 

When the bug happens, can you try on different iOS Devices?

**Comment by rmischook** [comment]
Yes I should have clarified that point 4 above referred to proxying another iPhone when the proxy stopped responding to the first. In that case the proxy ALSO was not working for the newly-added proxy. If I enabled LOCAL proxy - that DOES work for the Mac - but the proxy continues to ignore the iPhones.

**Comment by NghiaTranUIT** [comment]
It's hard to tell what causes the problem. It looks like your iOS Devices could not proxy their traffic to Proxyman Proxy Server at your IP address port 9090.

Just wondering: Are you using a corporate network or a home network? If it's a corporate network, from my experience, some company blocks it

**Comment by rmischook** [comment]
Home network. It can't be network config because IT WORKS FOR AWHILE and then JUST STOPS.

**Comment by NghiaTranUIT** [comment]
Thanks for your feedback @markusmattfolkstenberg. If you don't mind, can you try on different iOS devices when it occurs 🤔 

I haven't received any similar reports recently. I will keep an eye on it 👍
